pkg/sql/importer/import_job.go

```diff
@@ -129,21 +128,19 @@ func (r *importResumer) Resume(ctx context.Context, execCtx interface{}) error {
 					curDetails = schemaMetadata.schemaPreparedDetails
 				}
 
-				if r.settings.Version.IsActive(ctx, clusterversion.PublicSchemasWithDescriptors) {
-					// In 22.1, the Public schema should always be present in the database.
-					// Make sure it is part of schemaMetadata, it is not guaranteed to
-					// be added in prepareSchemasForIngestion if we're not importing any
-					// schemas.
-					// The Public schema will not change in the database so both the
-					// oldSchemaIDToName and newSchemaIDToName entries will be the
-					// same for the Public schema.
-					_, dbDesc, err := descsCol.GetImmutableDatabaseByID(ctx, txn, details.ParentID, tree.DatabaseLookupFlags{Required: true})
-					if err != nil {
-						return err
-					}
-					schemaMetadata.oldSchemaIDToName[dbDesc.GetSchemaID(tree.PublicSchema)] = tree.PublicSchema
-					schemaMetadata.newSchemaIDToName[dbDesc.GetSchemaID(tree.PublicSchema)] = tree.PublicSchema
+				// In 22.1, the Public schema should always be present in the database.
```

Comment:
This comment could be rewritten as "The public schema is expected to always be present in the database".